x-pack/plugins/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/span_flyout/sticky_span_properties.tsx

```diff
@@ -49,7 +50,9 @@ export function StickySpanProperties({ span, transaction }: Props) {
   });
 
   const spanName = span.span.name;
-  const dependencyName = span.span.destination?.service.resource;
+  // const dependencyName = span.span.destination?.service.resource;
+  const targetType = span.test?.span?.service?.target?.type || '';
```

Comment:
This needs to be updated to get the correct fields. See how we can get them from `span`